src/accordion.stories.ts

```diff
@@ -33,7 +33,7 @@ const meta: Meta = {
     context.canvasElement
       .querySelector('glide-core-accordion')
       ?.addEventListener('toggle', (event) => {
-        if (event.target instanceof GlideCoreAccordion) {
+        if (event.target instanceof AccordionComponent) {
```

Comment:
Suffixed with "Component" only in stories because there's a namespace conflict in each: 

```
export const Accordion: StoryObj = {
  tags: ['!autodocs'],
};
```